test/testutils.ts

```diff
@@ -172,6 +172,13 @@ export function decodeRevertReason (data: string, nullIfNoMatch = true): string
   } else if (methodSig === '0x4e487b71') {
     const [code] = ethers.utils.defaultAbiCoder.decode(['uint256'], dataParams)
     return `Panic(${panicCodes[code] ?? code} + ')`
+  } else if (methodSig === '0x8d6ea8be') {
+    const [reason] = ethers.utils.defaultAbiCoder.decode(['string'], dataParams)
+    return `CustomError("${reason as string}")`
```

Comment:
This `testutils` package was not meant to be a place for test-specific methods, but for "system-wide" reverts (yes, due to a lack of such a decoder in ethers itself)